nswag/Client.Class.ProcessResponse.liquid

```diff
@@ -68,5 +68,5 @@ return;
 {%     endif -%}
 {% else -%}{% comment %} implied: `if !response.HasType` so just read it as text {% endcomment %}
 string responseText_ = ( response_.Content == null ) ? string.Empty : await response_.Content.ReadAsStringAsync().ConfigureAwait(false);
-throw new {{ ExceptionClass }}("{{ response.ExceptionDescription }}", status_, responseText_, headers_, null);
+throw {{ ExceptionClass }}.Create(status_, headers_, responseText_, JsonSerializerSettings, null);
```

Comment:
This line handles the case when throwing a documented status code when there is no response body returned. This mostly only applies to `GetDocumentContentType`. An `ApiException` with a `ProblemDetails` with minimal information will be thrown. See this commit to view what changes this line made to the generated code.
https://github.com/Laserfiche/lf-repository-api-client-dotnet/pull/61/commits/730f08c4284a51131573adf3cc410aaa1065dd1a